src/test/test_tapyrus.cpp

```diff
@@ -57,10 +57,9 @@ BasicTestingSetup::BasicTestingSetup(const std::string& chainName)
     InitSignatureCache();
     InitScriptExecutionCache();
     fCheckBlockIndex = true;
-    assert(signedBlockCondition.threshold == 10);
-    assert(signedBlockCondition.pubkeys.size() == 15);
+    gArgs.SoftSetBoolArg("-server", true);
```

Comment:
What is this `-server` flag?

Reply:
This is a temporary workaround to allow the test process read "signblockpubkey" argument from command line. In PR#18 "signblockpubkey" argument is removed and this line is also removed.

This is a consequence of this change in src/chainparams.h

``` 
-  CChainParams():aggregatePubkey(GetAggregatePubkey()) {};

+  //require signedblockpubkey argument only in tapyrusd.  
+  CChainParams():aggregatePubkey(gArgs.GetBoolArg("-server", false) ?   GetAggregatePubkeyFromCmdLine() : CPubKey()) {}
```
before this change  "signblockpubkey" was mandatory in `tapyrusd` and `tapyrus-tx`as they  shares the same code.  But in 'tapyrus-tx` this argument is not required.